### ISSUE TITLE
Other implementation of the methods

### ISSUE THREAD

**Issue by 863689877** [sniklaus/softmax-splatting#43]
Thank you for your nice work!
I have a question whether it is necessary to use cupy to realize these splatting methods. If not, how can I realize these methods only using pytorch.
